conda/recipes/numbast_extensions/meta.yaml

```diff
@@ -17,13 +18,32 @@ source:
   path: ../../../numbast_extensions
 
 build:
+  number: {{ GIT_DESCRIBE_NUMBER }}
+  string: py{{ py_version }}_{{ GIT_DESCRIBE_HASH }}_{{ GIT_DESCRIBE_NUMBER }}
+  script_env:
+    - AWS_ACCESS_KEY_ID
+    - AWS_SECRET_ACCESS_KEY
+    - AWS_SESSION_TOKEN
+    - CMAKE_C_COMPILER_LAUNCHER
+    - CMAKE_CUDA_COMPILER_LAUNCHER
+    - CMAKE_CXX_COMPILER_LAUNCHER
+    - CMAKE_GENERATOR
+    - PARALLEL_LEVEL
+    - SCCACHE_BUCKET
+    - SCCACHE_IDLE_TIMEOUT
+    - SCCACHE_REGION
+    - SCCACHE_S3_KEY_PREFIX=numbast-aarch64 # [aarch64]
+    - SCCACHE_S3_KEY_PREFIX=numbast-linux64 # [linux64]
+    - SCCACHE_S3_USE_SSL
+    - SCCACHE_S3_NO_CREDENTIALS
   script:
     - {{ PYTHON }} -m pip install . -vv
 
 requirements:
   build:
     - python
     - pip
+    - setuptools
   run:
     - cuda-version >=12.5
```

Comment:
Should we change this to `${RAPIDS_CUDA_VERSION}`, otherwise this seems to be overriding the environment's CTK version.

Reply:
The same question goes to `conda/recipe/numbast/meta.yaml`.

Reply:
Currently Numbast test with the conda delivered CTK, and this PR should just keep the way it is (so that even though the environment has the site installed CTK, it is ignored). But still, it would be nice if there's a single source of truth. If the image is specified with 12.8, let's also install CTK 12.8 conda package.

Reply:
It might want a little more research to look at if this works as expected when installed in other conda envs, but for testing I think we are fine.

Reply:
I verified the tests are indeed running with CUDA 12.8 packages.